### PR TITLE
Update documentation for Windows PATH, fixes #94

### LINKED ISSUE #94: Improve documentation for makensis
Hi, I've installed nsis for windows, but still get 

```
- makensis not found.
  Please install "nullsoft scriptable install system"
  http://nsis.sourceforge.net/Download
```
Should I add it also to path?

### PR DESCRIPTION
**Pull request by mkaz**
Add info on setting Windows path to find `makensis`

### CODE REVIEW

**Comment by havenchyk** [comment]
Heh, cool, thanks. I just had going to add similar lines.

File: README.md [hunk 1]
```diff
@@ -52,6 +52,10 @@ On Ubuntu(-based) Linux distributions, via apt:
 # apt-get install wine nsis -y
 ```
 On Windows, download the [nullsoft scriptable installer](http://nsis.sourceforge.net/Download)
+You need to included NSIS in your PATH to find `makensis`, set your global enviornment variable or you can set a session variable using:
```
Comment (havenchyk) [inline README.md]: > You need to included NSIS
Typo: You need to include NSIS

**Comment by havenchyk** [comment]
LGTM

**Comment by stefanjudis** [comment]
:+1:

**Pull request merged**